.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,20 +1,25 @@
-<!--
-Please read and follow these instructions before creating and submitting a pull request:
+<!-- Thanks for contributing! -->
 
-- If you're fixing a bug, ensure you add unit tests to prove that it works.
-- Before adding a feature, it is best to create an issue explaining it first. It would save you some effort in case we don't consider it should be included in node-fetch.
-- If you are reporting a bug, adding failing units tests can be a good idea.
--->
+## Purpose
+...
 
-**What is the purpose of this pull request?**
+## Changes
+...
 
-- [ ] Documentation update
-- [ ] Bug fix
-- [ ] New feature
-- [ ] Other, please explain:
+## Additional information
+...
 
-**What changes did you make? (provide an overview)**
 
-**Which issue (if any) does this pull request address?**
+-------------------------------------------------------------------------------------------------
 
-**Is there anything you'd like reviewers to know?**
+<!-- Mark what you have done, Remove unnecessary ones. Add new tasks that may fit (like TODO's) -->
+- [ ] I prefixed the PR-title with `docs: `, `fix(area): `, `feat(area): ` or `breaking(area): `
+- [ ] I updated ./docs/CHANGELOG.md with a link to this PR or Issue
+- [ ] I updated ./docs/v3-UPGRADE-GUIDE
+- [ ] I updated the README.md
+- [ ] I added unit test(s)
+
+-------------------------------------------------------------------------------------------------
+
+<!-- Add a `- fix #_NUMBER_` line for every PR/Issue this PR solves. Don't comma separate them -->
```

Comment:
"do not" is a more formal way of saying "don't". Unless it is not what we're aiming for.

```suggestion
<!-- Add a `- fix #_NUMBER_` line for every PR/Issue this PR solves. Do not comma separate them -->
```

Reply:
> "do not" is a more formal way of saying "don't". Unless it is not what we're aiming for.

Can revert it. 

Could you explain what the differens is (by example)?  I'm not a native english speaker... I have Always thought they meant the exact same thing and it was just a shorter way of writing something.

Reply:
They do indeed mean the same thing but abbreviations have a reputation for being used in more informal contexts, or so my English teacher says.

Reply:
Thanks.